### DISCUSSION TITLE
[Feature] Ability to un tag a person from a photo

### DISCUSSION THREAD

**Discussion by aad4321** [immich-app/immich#11003] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a situation where the face detection is incorrect for a specific person. I dont see the option to unassign the face in the photo I am left with having to assign to another person. 

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by mschiff** [comment]
I am having a similar problem: Sometimes things get detected as faces. For some like statues this might be ok and I might name them. But there also sometime things where only a machine can see a face and which definitely is none.

What is missing here is a knob to say "This is not a face".

**Comment by mschiff** [comment]
Another note: A workaround might be to name a person like "No face" and assign all non-faces to it. But how will this influence the algorythm?
